components/Flashcard/Flashcard.js

```diff
@@ -4,13 +4,17 @@ import MarkAsIncorrect from "@/public/icons/MarkAsIncorrect.svg";
 import MarkAsCorrect from "@/public/icons/MarkAsCorrect.svg";
 import Delete from "@/public/icons/Delete.svg";
 import Arrow from "@/public/icons/Arrow.svg";
+import Edit from "@/public/icons/Edit.svg";
 import { RoundButton } from "../Button/Button";
 import DeleteConfirmationDialog from "../DeleteConfirmationDialog/DeleteConfirmationDialog";
 
 export default function Flashcard({
   flashcard,
   onToggleCorrect,
   handleDelete,
+  setCurrentFlashcard,
+  isEdit,
+  setIsEdit,
```

Comment:
💡 **Important:** Avoid drilling state setter functions `setCurrentFlashcard `and `isEdit`, as props. This is generally not recommended. It's better to wrap the state setter functions in a `handler function` within the parent component `App`. This approach encapsulates the logic within the parent, allowing you to maintain control over how the state is modified. 


_example for the isEdit state_
```js
function handleEdit() {
  setIsEdit((prevIsEdit) => !prevIsEdit);
}
```